okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -233,13 +234,28 @@ public Level getLevel() {
 
       if (!logBody || !HttpHeaders.hasBody(response)) {
         logger.log("<-- END HTTP");
-      } else if (bodyEncoded(response.headers())) {
+      } else if (bodyHasUnknownEncoding(response.headers())) {
         logger.log("<-- END HTTP (encoded body omitted)");
       } else {
         BufferedSource source = responseBody.source();
```

Comment:
Can this be simplified using response.peekBody earlier?  And also logging the bytes in the unknown encoding case?

Reply:
I'll check it out and see if it improves things.

Re logging the bytes in the unknown case - Sure - You mean the count of them, not the encoded (and presumably base64d) raw content, correct?

Reply:
yep, nothing blocking from me. Just asking the questions.

Reply:
Gave it a shot - https://github.com/mattsheppard/okhttp/pull/1/files - I'm not convinced it's better, but maybe I'm missing the point of your idea.

Happy to just add the logging number of bytes in the unknown case if you like in any case.